.travis.yml

```diff
@@ -41,5 +41,6 @@ install:
 
 script:
   - npm run build
-  - phpcs --standard=ruleset.xml --extensions=php --ignore=node_modules,bower_components,vendor -n -s .
+  - phpcs --report=summary
+  - phpcs --report=summary --standard=phpcs-templates.xml -n
```

Comment:
Note: `-n` means that travis will ignore phpcs warnings, only errors will trigger failures.

This gives us a bit of discretion.